com.unity.ml-agents/Runtime/Sensors/ICompressibleSensor.cs

```diff
@@ -0,0 +1,21 @@
+namespace Unity.MLAgents.Sensors
+{
+    /// <summary>
+    /// Sensor interface for generating observations.
+    /// </summary>
+    public interface ICompressibleSensor : ISensor
```

Comment:
I still think we need a different name for this, since sensors that don't implement the interface can still be compressible. I suggested `ISparseChannelSensor` in another comment; I don't love that but I like it better than the current form.